Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "wsts"
-version = "9.1.0"
+version = "10.0.0"
```

Comment:
If we're going to bump the version, should we remove that unused error variant?

Reply:
Yeah we can go wild here.

Reply:
Done